src/components/Error/Error.tsx

```diff
@@ -0,0 +1,34 @@
+import classNames from 'classnames';
+import { ErrorType } from '../../types/ErrorType';
+
+type Props = {
+  onErrorDisable: () => void,
+  errorType: ErrorType,
+};
+
+export const Error: React.FC<Props> = ({
+  onErrorDisable,
+  errorType,
+}) => {
+  return (
+    <div
+      data-cy="ErrorNotification"
+      className={classNames(
+        'notification',
+        'is-danger',
+        'is-light',
+        'has-text-weight-normal',
+        { hidden: errorType === ErrorType.success },
+      )}
+    >
+      <button
+        aria-label="hide_error"
+        data-cy="HideErrorButton"
+        type="button"
+        className="delete"
+        onClick={() => onErrorDisable}
```

Comment:
the onClick won't be called
```suggestion
        onClick={onErrorDisable}
```